src/search-component/search-component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './search.css';
+
+class SearchComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { searchValue: '', addedText: '' };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleButtonAction = this.handleButtonAction.bind(this);
+  }
+
+  handleInputChange(event) {
+    this.setState({ searchValue: event.target.value });
+  }
+
+  handleButtonAction() {
+    const { searchValue, addedText } = this.state;
+    this.setState({ addedText: `${addedText} ${searchValue}, ` });
+  }
+
+  render() {
+    const { searchValue, addedText } = this.state;
+    return (
+      <div className="search-container">
+        <input type="text" value={searchValue} onChange={this.handleInputChange} />
```

Comment:
it's not important for this task and module, but it's good to use either <label> tag with attribute `htmlFor` or `aria-label` tag in input for accessibility purpose